cpp/cmake/thirdparty/get_raft.cmake

```diff
@@ -30,8 +30,8 @@ function(find_and_configure_raft)
             BUILD_EXPORT_SET    cuml-exports
             INSTALL_EXPORT_SET  cuml-exports
             CPM_ARGS
-            GIT_REPOSITORY https://github.com/${PKG_FORK}/raft.git
-            GIT_TAG        ${PKG_PINNED_TAG}
+            GIT_REPOSITORY https://github.com/robertmaynard/raft.git
+            GIT_TAG        refactor_cmake_raft_target_logic
```

Comment:
Changes look great, just leaving the obligatory comment to change this back to main